dbscripts/xml/upgrade.xml

```diff
@@ -149,6 +149,7 @@
 		<migration class="APP\migration\upgrade\v3_5_0\I9937_EditorialTeamToEditorialHistory"/>
 		<migration class="PKP\migration\upgrade\v3_5_0\I10041_UserGroupsAndUserUserGroupsMastheadValues"/>
 		<migration class="PKP\migration\upgrade\v3_5_0\I9771_OrcidMigration"/>
+		<migration class="APP\migration\upgrade\v3_5_0\I10157_CreateAndMigrateIssueIdField.php"/>
```

Comment:
This doesn't appear to be working; see the [test failure on the upgrade test](https://github.com/pkp/ojs/actions/runs/13537527205/job/37831790207?pr=4396):
![image](https://github.com/user-attachments/assets/08048b38-2a00-4c46-84cf-6e13ec21b3a1)